webapp/models.py

```diff
@@ -185,29 +133,13 @@ def cves_ids(self):
         return [cve.id for cve in self.cves]
 
     @hybrid_property
-    def package_list(self):
-        if not self.release_packages:
-            return []
-
-        package_list = []
-        for codename, packages in self.release_packages.items():
-            for package in packages:
-                package_list.append(package["name"])
-
-        return set(sorted(package_list))
-
-    def as_dict(self):
-        return {
-            "id": self.id,
-            "title": self.title,
-            "published": self.published,
-            "summary": self.summary,
-            "details": self.details,
-            "instructions": self.instructions,
-            "references": self.references,
-            "release_packages": self.release_packages,
-            "cves": [c.id for c in self.cves],
-        }
+    def type(self):
```

Comment:
I don't know if you are having any errors, I suppose not, but this could be overriding an existent [python function](https://docs.python.org/3/library/functions.html#type)

Reply:
I renamed it to "notice_type" to avoid any possible complications.